src/coffea/util.py

```diff
@@ -98,6 +99,25 @@ def _ensure_flat(array, allow_missing=False):
     return array
 
 
+def _getaxis(name, var, bins, start, stop, edges, transform, delayed_mode):
```

Comment:
more descriptive function name here please - there are at least three kinds of axes I can think of.